core/services/gateway/handlers/functions/handler.functions.go

```diff
@@ -133,7 +133,7 @@ func (h *functionsHandler) HandleUserMessage(ctx context.Context, msg *api.Messa
 }
 
 func (h *functionsHandler) handleSecretsRequest(ctx context.Context, msg *api.Message, callbackCh chan<- handlers.UserCallbackPayload) error {
-	h.lggr.Debugw("handleSecretsRequest: processing message", "sender", msg.Body.Sender)
+	h.lggr.Debugw("handleSecretsRequest: processing message", "sender", msg.Body.Sender, "id", msg.Body.MessageId)
```

Comment:
Minor: "id" => "messageId", for clarify, otherwise easy to mess with requestId or other IDs.